FND/GoalsAndObjectives/Objectives.rdf

```diff
@@ -109,6 +110,17 @@
 		<skos:definition>short-term, quantitative, measurable result that a person, organization, or system seeks to attain in order to achieve its long-term goals</skos:definition>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-fnd-gao-obj;Program">
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-gao-obj;hasObjective"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-gao-obj;Objective"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>program</rdfs:label>
```

Comment:
this should be marked as lang en/us. en/gb would be programme

Reply:
FIBO is an American standard, and should not need the English reference, but ok.

Reply:
No FIBO is meant to be international not solely American.